### PR TITLE
Add Faker.Pokemon.It.name()

### PR DESCRIPTION
**Pull request by fusillicode**
I've added:

- [x] Faker.Pokemon.It.name/0

I've updated:

- [x] USAGE.md docs
- [x] CHANGELOG.md

[Open Source Saturday](https://www.meetup.com/it-IT/Open-Source-Saturday-Milano/)

P.S: I took the liberty to add ☝️ badge. Let me know if that's ok for you 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Current list of maintainers:
 
 * [Anthony Smith](https://github.com/anthonator)
 * [Igor Kapkov](https://igas.me)
+* [Gian D](https://github.com/fusillicode)
```
Comment (vbrazo) [inline README.md]: this is the list of maintainers, not the contributors.

Comment (fusillicode) [reply README.md]: Oh gosh thanks for pointing out! 🙇 

It should be fixed now! 😄

**Comment by fusillicode** [comment]
Thank you for all your work on this library! 😄 🙇

**Comment by vbrazo** [comment]
@fusillicode No worries! We do our best. 

Feel free to complete the missing Italian translations. They're more welcome here :)

**Pull request merged**